crates/bevy_internal/Cargo.toml

```
@@ -98,4 +98,4 @@ bevy_winit = { path = "../bevy_winit", optional = true, version = "0.8.0-dev" }
 bevy_gilrs = { path = "../bevy_gilrs", optional = true, version = "0.8.0-dev" }
 
 [target.'cfg(target_os = "android")'.dependencies]
-ndk-glue = {version = "0.6", features = ["logger"]}
+ndk-glue = {version = "0.5", features = ["logger"]}
```

Same comment regarding spaces here?

```suggestion
ndk-glue = { version = "0.5", features = ["logger"] }
```

---

Eh I don't mind it as much here; the other one was imbalanced.

---

It's very inconsistent with the rest of the file :see_no_evil:, but you do you...